pkg/services/alerting/notifiers/pagerduty.go

```diff
@@ -81,8 +81,17 @@ func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
 		eventType = "resolve"
 	}
 	customData := triggMetrString
+
+	evalMatches := simplejson.New()
 	for _, evt := range evalContext.EvalMatches {
 		customData = customData + fmt.Sprintf("%s: %v\n", evt.Metric, evt.Value)
+		evalMatches.Set("eval_metric", evt.Metric)
+		evalMatches.Set("eval_value", evt.Value)
+		if len(evt.Tags) > 0 {
```

Comment:
I think this conditional is not necessary; in the case there are no tags it will not enter to the loop anyway.

Reply:
To me it seems like evalMatches is being overwritten if there are multiple `evalContext.EvalMatches`?